builders/xcm/fees.md

```diff
@@ -180,52 +190,58 @@ As an example, you can calculate the total cost of KSMs for sending an XCM messa
 
-=== "Moonbeam"
-    |                                                                                                   Weight                                                                                                    |                                                                            Wei per weight                                                                            |
-    |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
-    | [{{ networks.moonbeam.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonbeam/src/xcm_config.rs#L201){target=_blank} | [{{ networks.moonbeam.xcm.instructions.wei_per_weight.display }}](https://github.com/PureStake/moonbeam/blob/master/runtime/moonbeam/src/lib.rs#L128){target=_blank} |
+The following sections will break down how to calculate XCM fees for Moonbeam-based networks. There are two main scenarios:
 
-=== "Moonriver"
-    |                                                                                                    Weight                                                                                                     |                                                                            Wei per weight                                                                             |
-    |:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
-    | [{{ networks.moonriver.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonriver/src/xcm_config.rs#L208){target=_blank} | [{{ networks.moonriver.xcm.instructions.wei_per_weight.display }}](https://github.com/PureStake/moonbeam/blob/master/runtime/moonbeam/src/lib.rs#L128){target=_blank} |
+1. Fees paid in the reserve token (native tokens like GLMR, MOVR, or DEV)
+2. Fees paid in external assets (XC-20s)
```

Comment:
Not sure what you think, but I feel like we typically only use numbers for steps and bullet points for other lists... either way not really a big deal

Reply:
Yeah sure, you are more aware of the rules than me 😃